src/tests.rs

```diff
@@ -1,3 +1,10 @@
+#[cfg(test)]
```

Comment:
Why do we need the cargo tests that run the interop tests at all.

Reply:
I decided to implement them because it made testing changes during development a lot quicker and more convenient, and because the standard configuration of travis-CI for rust simply runs cargo test.
I realized that there is now basically two ways to achieve the same result, running 'cargo test' or running interop the way it's supposed to be run. Maintaining both options doesn't seem useful, and the run.sh script actually makes the latter almost as convenient as 'cargo test', so I'll focus on improving that instead of the internal rust tests.